build/generateTsDefs.py

```diff
@@ -102,6 +102,8 @@ def GenerateTsDefs(inputs, output):
     f.write(license_header)
     f.write(b'\n')
     f.write(contents)
+    f.write(b'\n')
+    f.write('export default shaka;'.encode(encoding='utf-8'))
```

Comment:
Notice that we use byte-strings for the other literals.  Just use `b'export default shaka;\n'`